### DISCUSSION TITLE
Calculated Miller indices different from shown in IPF

### DISCUSSION THREAD

**Discussion by johnyejungu** [mtex-toolbox/mtex#536] [Ask Anything] [answered]
I am trying to use mtex to generate a set of grain orientations and find the corresponding Miller indices. 
However, the results calculated according to the documentation (https://mtex-toolbox.github.io/OrientationInversePoleFigure.html) is different from that shown in the IPF. There must be something missing for the 1-on-1 comparison. The details are shown in the attached screenshot. 
Thanks if anyone can share their experience!
![mtex](https://user-images.githubusercontent.com/7401416/96665359-b87dcc80-1322-11eb-9826-bd1d88357135.png)

**Comment by kilir** [comment, marked as answer]
Hi, 
the plotIPDF shows the fundamental sector. If you just use plot(h), you may see the difference. plotIPDF() uses
https://mtex-toolbox.github.io/Miller.project2FundamentalRegion.html

Hope that helps.
Cheers Rüdiger
